invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -140,13 +140,15 @@ <h1 class="ui medium header mb-0">{{ community.metadata.title }}</h1>
     <div
       class="ui container secondary pointing stackable menu page-subheader pl-0 pr-0">
       {% for item in current_menu.submenu('communities').children if item.permissions == True or permissions[item.permissions] %}
+        {% if item.visible %}
```

Comment:
```suggestion
      {% for item in current_menu.submenu('communities').children if item.permissions == True or permissions[item.permissions] and item.visible %}
```

minor/nit: you can add it to the for loop `if ...` clause also to avoid the extra nesting :)